internal/providers/terraform/google/storage_bucket.go

```diff
@@ -194,142 +205,121 @@ func dataRetrieval(d *schema.ResourceData, u *schema.UsageData) *schema.CostComp
 	}
 }
 
-// TODO: Move to a separate resource, similar to what we did for AWS, also needs other data transfer prices
-
-// infracost-usage-example.yml for this
```

Comment:
this should go back into that file

Reply:
A separate file for storage bucket or a separate file for whole google data transfer and a function for storage bucket?

Reply:
sorry, I meant the infracost-usage-example.yml file should be updated as part of this PR to add the keys/descriptions (I had moved the values here to avoid confusing users)

Reply:
Sure, I missed committing it.